skyportal/handlers/api/comment.py

```diff
@@ -48,9 +57,29 @@ def get(self, comment_id):
               application/json:
                 schema: Error
         """
-        comment = Comment.get_if_accessible_by(
-            comment_id, self.current_user, raise_if_none=True
-        )
+
+        if comment_on_what is None:
+            comment_on_what = 'object'
```

Comment:
```suggestion
```
Don't need this if you set the default in the function signature

Reply:
Yeah I tried this but it doesn't work. There's something with the handler's default parameters and the regexp that parses the path parameters that just didn't work (it put a None in there, despite having a default parameter...)

Reply:
Ah okay, sure no problem then